### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by myelin0**
## In this branch I add the Routes for the SPA and add basic styling to each.

- Use React Router.
- Use styles in a React app.
- Pass Linters.

### CODE REVIEW

File: src/Quotes.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      <div className="btnContainer">
+        <button
+          type="button"
+          className="btn btn-warning"
+          onClick={getQuotes}
+        >
+          Get Quote
+        </button>
+        <a
+          href={`http://twitter.com/intent/tweet?text=${quotes.text}`}
```
Comment (micheaol) [inline src/Quotes.js:45]: Please ensure that the general layout should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).

Please `align center` as in the original design. 

Your Implementation | Design
:---:|:---:
image | image


Your Implementation | Design
:---:|:---:
image | image

Please also ensure to include `footer` in all the pages according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @myelin0 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/Quotes.js (hunk 1 above)
Comment (myelin0) [reply src/Quotes.js:45]: Hi @micheaol ! Thank you for your review, but I wanted to ask you about the footer. I don`t read anywhere that we have to create a footer for this project. Can you confirm this is optional? Thank you in advance!

Comment (micheaol) [reply src/Quotes.js:45]: @myelin0,

The project requirement states that your implementation should match the given wireframes in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md). which includes everything in the wireframe.

I hope this helps.

**Review by mmsesay — APPROVED** [review]
Hi @myelin0, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**